worksheet/index.qmd

```diff
@@ -0,0 +1,137 @@
+---
+format: typst
+---
+
+```{=typst}
+#set page(
+  flipped: true, 
+  columns: 2, 
+  margin: (x: 0.5in, y: 1in),
+  numbering: none
+)
+
+#set list(marker: ([#sym.square.stroked], [-]))
+```
+
+## Writing your first documentation PR
+
+The goal of this worksheet is to guide you through the steps needed to successfully create a documentation PR (pull request). Empty spaces have been placed on this sheet for your notes.
+
+### 0. Get ready
+
+- `install.packages("pak")`
+
+- `pak::pak("devtools")`
+
+- .Set up your GitHub personal access token.
+    - `usethis.r-lib.org` -> Articles -> Managing Git(Hub) Credentials
+
+- Call `usethis::git_sitrep()` and check that:
+    - Your name and email address appears in "Git config (global)"
+    - Your GitHub user name is found under "Github"
```

Comment:
Not sure if we want to mention this explicitly? But one of the most common mistakes I see at this stage is people don't use `usethis::create_github_token()` and when they get their PAT, they don't pick the right scopes. That will also generate a "tsk, tsk!" from `git_sitrep()`.

We might also want to recommend checking their "git vaccination status" and dealing with it.

Reply:
i will refer to you, as you have helped many more people deal with git then I ever will